docs/Usage.rst

```diff
@@ -75,22 +75,34 @@ This is done by first creating the necessary training data (based on the trainin
 
 .. code-block:: python
 
-    ml_pred = mqt.predictor.ml.Predictor()
+    ml_pred = mqt.predictor.ml.Predictor(figure_of_merit="expected_fidelity", devices="all")
```

Comment:
Similar to the comment in the README I'd argue that the `devices` parameter should be `str | None` and default to `None`, which would imply that all devices are selected.

Reply:
I agree and adjusted it.